driver-sync/src/test/functional/com/mongodb/client/AbstractSessionsProseTest.java

```diff
@@ -55,9 +57,25 @@
 public abstract class AbstractSessionsProseTest {
 
     private static final int MONGOCRYPTD_PORT = 47017;
+    private static Process mongocryptdProcess;
 
     protected abstract MongoClient getMongoClient(MongoClientSettings settings);
 
+    @BeforeAll
+    public static void beforeAll() throws IOException {
+        if (serverVersionAtLeast(4, 2)) {
+            mongocryptdProcess = startMongocryptdProcess();
+        }
+    }
+
+    @AfterAll
+    public static void afterAll() {
+        if (mongocryptdProcess != null) {
+            mongocryptdProcess.destroy();
```

Comment:
Given the empirical analysis shows no test interference across four runs, I still suggest waiting for the process to exit with `mongocryptdProcess.waitFor(TIMEOUT_DURATION.toMillis(), TimeUnit.MILLISECONDS); ` for extra certainty.

Reply:
I'm going to hold off on this unless it proves necessary.